httpapi/httpapi_test.go

```diff
@@ -58,3 +61,45 @@ func TestHTTPAPISybils(t *testing.T) {
 		}
 	}
 }
+
+func TestHTTPAPIRecordsListWithoutRecords(t *testing.T) {
+	ds := datastore.NewMapDatastore()
```

Comment:
Can we put a fake entry directly in the datastore to retrieve?

Reply:
@alanshaw once we have the ProviderManager, then we quickly can do a pm.AddProvider. Was waiting for that, sounds good?

Reply:
Ok sure!